.github/workflows/ci.yml

```diff
@@ -0,0 +1,106 @@
+name: CI
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  macos_debug_build_test:
+    runs-on: macos-10.15
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Switch to Xcode 11.2.1
+      run: sudo xcode-select --switch /Applications/Xcode_11.2.1.app/Contents/Developer
+    - name: macOS Debug Build and Test
+      run: |
+        source scripts/xcodebuild_wrapper.sh
+        macos_build OfficeUIFabricTestApp Debug build test
+  
+  macos_release_build_test:
+    runs-on: macos-10.15
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Switch to Xcode 11.2.1
+      run: sudo xcode-select --switch /Applications/Xcode_11.2.1.app/Contents/Developer
+    - name: macOS Release Build and Test
+      run: |
+        source scripts/xcodebuild_wrapper.sh
+        macos_build OfficeUIFabricTestApp Release build test
+
+  ios_simulator_debug_build_test:
+    runs-on: macos-10.15
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Switch to Xcode 11.2.1
+      run: sudo xcode-select --switch /Applications/Xcode_11.2.1.app/Contents/Developer
+    - name: iOS Simulator Debug Build and Test
+      run: |
+        source scripts/xcodebuild_wrapper.sh
+        ios_simulator_build OfficeUIFabric Debug build test
+  
+  ios_simulator_release_build:
+    runs-on: macos-10.15
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Switch to Xcode 11.2.1
+      run: sudo xcode-select --switch /Applications/Xcode_11.2.1.app/Contents/Developer
+    - name: iOS Simulator Release Build
+      run: |
+        source scripts/xcodebuild_wrapper.sh
+        ios_simulator_build OfficeUIFabric Release build
+
+  ios_testapp_simulator_debug_build:
```

Comment:
testapp before ios? Per another comment it feels like it is in between two "platform" tokens

Reply:
Switched the naming to the specific build command we're running to make it easier for developers to match the behavior on their local box.